manifest.yml

```diff
@@ -0,0 +1,11 @@
+---
+applications:
+- name: hackmd
+  instances: 1
+  memory: 2G
+  buildpack: nodejs_buildpack
+  env:
+    PGSSLMODE: require
+    UV_THREADPOOL_SIZE: 100
+    CMD_PORT: 8080
+    CF_DB: hackmd-db
```

Comment:
I think specifying the DB name really belongs in the deployment pipeline as that's what creates/names the database.

Reply:
I think you are right. I avoided it because it would have involved using some scripting to change the manifest within the Concourse task. I can change it if you want me to.